app/controllers/users_controller.rb

```diff
@@ -29,7 +30,8 @@ def update
     @user.update(user_params)
     if @user.save
       flash[:notice] = "Thine information has been updated!"
-      redirect_to dashboard_path
+      redirect_to dashboard_path if current_user.default?
```

Comment:
Thanks for the merge Matt. Upon 1 more private meeting we've decided you will not be penalized for the obstruction.